packages/atomic/cypress/integration/facet-date.cypress.ts

```diff
@@ -0,0 +1,233 @@
+  </atomic-date-facet>`
+  );
+}
+
+function setupCustomDateFacet(
+  field: string,
+  label: string,
+  dateRange: DateRanges
+) {
+  setUpPage(
+    `<atomic-breadcrumb-manager></atomic-breadcrumb-manager>
+    <atomic-date-facet field="${field}" label="${label}">
+           <atomic-date-range start=${dateRange.range1.start} end=${dateRange.range1.end} end-inclusive="true"></atomic-date-range>
+           <atomic-date-range start=${dateRange.range2.start} end=${dateRange.range2.end} end-inclusive="true"></atomic-date-range>
+           <atomic-date-range start=${dateRange.range3.start} end=${dateRange.range3.end} end-inclusive="true"></atomic-date-range>
+  </atomic-date-facet>`
+  );
+}
+
+describe('Standard Numeric Facet', () => {
+  beforeEach(() => {
+    setupAutoDateFacet(dateFacetProp.field, dateFacetProp.label);
+    createAliasShadow(dateFacetProp.field, FacetSelectors.facetDate);
+    createAliasFacetUL(dateFacetProp.field, FacetSelectors.facetDate);
+  });
+
+  describe('When page is loaded', () => {
+    it('Numeric facet should load, pass accessibility test and have correct label and facetCount', () => {
+      validateFacetComponentLoaded(
+        dateFacetProp.label,
+        FacetSelectors.facetDate
+      );
+    });
+
+    it('Should contains more than 1 checkboxes', () => {
+      validateFacetNumberofValueGreaterThan(1);
+    });
+
+    describe('When select 1 facetValue checkbox', () => {
+      it('Should activate checkbox, have valid facetCount and log UA', () => {
+        assertBasicFacetFunctionality('@firstFacetValue', dateFacetProp.field);
+      });
+
+      it('Should trigger breadcrumb and display correctly', () => {
+        cy.get('@firstFacetValue').click();
+        createBreadcrumbShadowAlias();
+        cy.get('@breadcrumbClearAllFilter').should('be.visible');
+        facetValueShouldDisplayInBreadcrumb(
+          '@firstFacetValue',
+          '.breadcrumb:nth-child(1) button span'
+        );
+      });
+
+      it('Should reflect selected facetValue on URL', () => {
+        cy.get('@firstFacetValue')
+          .click()
+          .find('label span:nth-child(1)')
+          .invoke('text')
+          .then((txt) => {
+            const facetValueInApiFormat = convertFacetValueToAPIformat(
+              txt,
+              FacetType.dateFacet
+            );
+            const urlHash = `#df[${dateFacetProp.field}]=${encodeURI(
+              facetValueInApiFormat
+            )}`;
+            cy.url().should('include', urlHash);
+          });
+      });
+    });
+
+    describe('When deselect 1 selected facetValue checkbox', () => {
+      it('should clear the checkbox and log UA', () => {
+        assertDeselectFacet(dateFacetProp.field);
+      });
+    });
+
+    describe('When click ClearAll facet', () => {
+      it('Should clear all checkboxes and log UA', () => {
+        assertClearAllFacet();
+      });
+    });
+  });
+});
+
+describe('Date facet with custom ranges', () => {
```

Comment:
I'd prefer to use consistent language to name things. In this case, using `manually specified` instead of `custom` ranges.

```
Date facet with manually specified ranges
```